opentech/apply/funds/management/commands/seed_rapid_response.py

```diff
@@ -0,0 +1,109 @@
+            {"type": "text_markup", "value": "<h3>I acknowledge</h3>"},
+            {"type": "checkbox", "value": {"field_label": "My application will be dismissed if it does not fit within OTF\'s mission, values, principles statements.", "help_text": "", "default_value": "true"}},
+            {"type": "text_markup", "value": "Read our <a href=\"\/about/program\">mission, values, and principles</a>."},
+            {"type": "checkbox", "value": {"field_label": "I have read and understand OTF\'s Terms and Privacy policy.", "help_text": "", "default_value": "true"}},
+            {"type": "text_markup", "value": "Read the <a href=\"\/tos\">Terms and Privacy policy</a>."},
+            {"type": "checkbox", "value": {"field_label": "I am legally able to sign contracts or represent an organization that can.", "help_text": "", "default_value": "true"}},
+            {"type": "checkbox", "value": {"field_label": "I understand that all intellectual property created with support for this application must be openly licensed.", "help_text": "", "default_value": "true"}},
+            {"type": "checkbox", "value": {"field_label": "I understand that if my application is incomplete in any way, it will be dismissed.", "help_text": "", "default_value": "true"}},
+            {"type": "checkbox", "value": {"field_label": "I understand that if my application is after a deadline, it will not be reviewed until after the next deadline.", "help_text": "", "default_value": "true"}},
+            {"type": "text_markup", "value": "<h3>I would like to</h3>"},
+            {"type": "checkbox", "value": {"field_label": "Sign up to the OTF-Announce list, low traffic (funding opportunities, major alerts, etc)", "help_text": "", "default_value": "false"}},
+            {"type": "checkbox", "value": {"field_label": "Sign up for OTF\'s daily newsletter (collection of news related to global internet freedom).", "help_text": "", "default_value": "false"}}
+        ])})
+
+        return application_form
+
+    def create_rapid_response_fund_type(self, application_form):
+        try:
+            fund = FundType.objects.get(title='Rapid Response')
+        except FundType.DoesNotExist:
+            apply_home = ApplyHomePage.objects.first()
+
+            fund = FundType(title='Rapid Response', workflow_name='single')
+            apply_home.add_child(instance=fund)
+
+            fund_form = FundForm.objects.create(fund=fund, form=application_form)
+            fund.forms = [fund_form]
+            fund.save()
+
+        return fund
+
+    def create_rapid_response_round(self, fund):
+        User = get_user_model()
+        lead_qs = User.objects.filter(full_name="Lindsay Beck")
+        lead = lead_qs.first() if lead_qs.count() else User.objects.filter(groups__name=STAFF_GROUP_NAME).first()
+
+        round = Round(
+            title="Rapid Response open round",
+            lead=lead,
+            start_date=date(2015, 8, 28),
```

Comment:
Why this date?

Reply:
That is the date of the original Rapid Response request type

Reply:
Commented